### DISCUSSION TITLE
Job Fails With Orphaned Error

### DISCUSSION THREAD

**Discussion by itizarsa** [jobrunr/jobrunr#1018] [General]
I went through the discussion on #654, but we aren't modifying any jobrunr internals on our job.

The job processes a big JSONL file, we read the file using the input stream & using the scanner to read line by line & update it to the DB. We had a job that took 30 minutes to run without any issues, but jobs exceeding 30 minutes failed with that error.

I'm using SQLServer & JpaRepository to save those rows. 

What could have gone wrong here? How do we prevent the orphaned error?

**Comment by rdehuyss** [comment]
Hi @itizarsa , have you checked the first page of the dashboard? It will mention there are problems regarding either CPU Allocation and I'm afraid this is out of our control (do you perhaps have a memory leak? Did you try a profiler)... 

You can increase the pollIntervalSize but it won't solve the root cause of the problem which is on your end.

**Reply by itizarsa** [reply]
The dashboard doesn't have any warnings. I have made a few changes to the code & yet to face the error again. I'll run the profiler too. Thanks
